diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -1853,6 +1857,17 @@ int DifferentialFunctionComparator::cmpOperationsWithOperands(
     RETURN_WITH_LOG(0);
 }
 
+int DifferentialFunctionComparator::cmpOperandTypes(
+        const llvm::Instruction *L, const llvm::Instruction *R) const {
+    if (int Res = cmpNumbers(L->getNumOperands(), R->getNumOperands()))
+        return Res;
+    for (unsigned i = 0, e = L->getNumOperands(); i != e; ++i)
+        if (int Res = cmpTypes(L->getOperand(i)->getType(),
+                               R->getOperand(i)->getType()))
+            return Res;
```

Comment:
We prefer braces around multi-line loop bodies (even if it's a just single command):
```suggestion
    for (unsigned i = 0, e = L->getNumOperands(); i != e; ++i) {
        if (int Res = cmpTypes(L->getOperand(i)->getType(),
                               R->getOperand(i)->getType()))
            return Res;
    }
```